app/scripts/controller-init/confirmations/transaction-controller-init.ts

```diff
@@ -1,5 +1,6 @@
 import {
   CHAIN_IDS,
+  type PublishBatchHookRequest,
```

Comment:
This one will be available once the TransactionController changes are merged and used in the extension.